core/src/main/scala/com/acervera/osm4scala/model/model.scala

```diff
@@ -88,7 +89,9 @@ case class WayEntity(
     id: Long,
     nodes: Seq[Long],
     tags: Map[String, String],
-    info: Option[Info] = None
+    info: Option[Info] = None,
+    lat: Seq[Double],
```

Comment:
What do you think about adding default values as Seq.empty to simplify migration to the new version? That will do other users happier than if they need to deal with compilation errors. ;)

Reply:
added